about.html

```diff
@@ -12,60 +12,84 @@
+     <!-- navigation bar and logo -->
+     <nav>
+        <div class="navLinks">
+            <img class="logo" src="img/lambda-black.png" alt="Lambda logo">
+            <a href="index.html">Home</a>
+            <a href="#about">About</a>
+            <a href="#products">Products</a>
+            <a href="#blog">Blog</a>
+            <a href="contact.html">Contact</a>
+        </div>
+    </nav>
 
+        <!-- header image -->
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
+        <img class="aboutHeaderImage" src="img/jumbo-about.png" alt="skyline">
 
+    <div class="container-about-page">
+        <section class="about-top-content">
+            <div class="about-text-container">
+              <h3>About</h3>
+              <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+            </div>
+        </section>
 
 
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
 
-    Learn More
 
+    <!-- grid middle -->
 
+    <section id="aboutMain">
+      <div class="aboutGrid">
+        <img src="img/about-plan.png" alt="strategy">
+        <h2>Strategy</h2>
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <a href="#">Learn More</a>
+      </div>
 
-    <img src="img/about-office.png" alt="strategy">
+      <div class="aboutGrid">
+        <img src="img/about-working.png" alt="strategy">
+        <h2>How We Work</h2>
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <a href="#">Learn More</a>
+        
+      </div>
     
-    Places We Work
+      <div class="aboutGrid">
+       <img src="img/about-office.png" alt="strategy">
+        <h2>Places We Work</h2>
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <a href="#">Learn More</a>
+        
+      </div>
 
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+      <div class="aboutGrid">
```

Comment:
Love the consistent "grid" classes to maintain styles amongst each item. Awesome!